models/marts/customer_metrics.sql

```diff
@@ -0,0 +1,13 @@
+{{
+    config(
+        schema=var('customer')
+    )
+}}
+
+select
+    -- your logic
+    *
+from
+    {{ ref('customer_data') }}
+where
+    customer_id = '{{ var('customer') }}'
```

Comment:
What are we looking to demo with customer_data + customer_metrics?

Reply:
Yeah this is not necessary, I'll remove this!